shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -298,13 +297,21 @@ - (void)updateWindowMetrics {
   CGSize scaledSize = [view convertRectToBacking:view.bounds].size;
   double pixelRatio = view.bounds.size.width == 0 ? 1 : scaledSize.width / view.bounds.size.width;
 
-  const FlutterWindowMetricsEvent event = {
-      .struct_size = sizeof(event),
+  // The screen is currently always the same size as the window.
```

Comment:
Can we reference an issue that describes a transition plan for this? It seems weird to intentionally send incorrect information without more detail about the plan.